specification/applicationinsights/resource-manager/Microsoft.Insights/preview/2019-10-01-preview/WebTestResults_API.json

```diff
@@ -0,0 +1,976 @@
+{
+	"swagger": "2.0",
+	"info": {
+		"title": "MonitorManagementClient",
+		"x-ms-code-generation-settings": {
+			"name": "MonitorManagementClient"
+		},
+		"version": "2019-10-01-preview"
+	},
+	"host": "management.azure.com",
+	"schemes": [
+		"https"
+	],
+	"consumes": [],
+	"produces": [
+		"application/json"
+	],
+	"security": [
+		{
+			"azure_auth": [
+				"user_impersonation"
+			]
+		}
+	],
+	"securityDefinitions": {
+		"azure_auth": {
+			"type": "oauth2",
+			"authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+			"flow": "implicit",
+			"description": "Azure Active Directory OAuth2 Flow",
+			"scopes": {
+				"user_impersonation": "impersonate your user account"
+			}
+		}
+	},
+	"paths": {
+		"/subscriptions/{subscriptionId}/resoucegroups/{resourceGroupName}/providers/microsoft.insights/webtests/{webtestName}/popbrs/testresultfile": {
+			"get": {
+				"tags": [
+					"FindWebTestResult"
```

Comment:
Should this be under the "WebTests" tag to associate all the web test related methods?